modules/react/modal/stories/examples/Basic.tsx

```diff
@@ -1,32 +1,37 @@
 import React from 'react';
 
 import {Modal} from '@workday/canvas-kit-react/modal';
-import {DeleteButton} from '@workday/canvas-kit-react/button';
+import {PrimaryButton} from '@workday/canvas-kit-react/button';
 import {HStack, Box} from '@workday/canvas-kit-react/layout';
 
 export const Basic = () => {
-  const handleDelete = () => {
-    console.log('Deleted item');
+  const handleAcknowledge = () => {
+    console.log('License Acknowledged');
+  };
+
+  const handleCancel = () => {
+    console.log('Cancel clicked');
   };
 
   return (
     <Modal>
-      <Modal.Target as={DeleteButton}>Delete Item</Modal.Target>
+      <Modal.Target as={PrimaryButton}>Open License</Modal.Target>
       <Modal.Overlay>
         <Modal.Card>
           <Modal.CloseIcon aria-label="Close" />
-          <Modal.Heading>Delete Item</Modal.Heading>
+          <Modal.Heading>MIT License</Modal.Heading>
           <Modal.Body>
             <Box as="p" marginY="zero">
-              Are you sure you want to delete the item?
+              Permission is hereby granted, free of charge, to any person obtaining a copy of this
+              software and associated documentation files (the "Software").
             </Box>
+            <HStack spacing="s">
+              <Modal.CloseButton as={PrimaryButton} onClick={handleAcknowledge}>
+                Acknowledge
+              </Modal.CloseButton>
+              <Modal.CloseButton onClick={handleCancel}>Cancel</Modal.CloseButton>
+            </HStack>
```

Comment:
Let's change this back to live outside of `Modal.Body` and have the same padding and marginTop as before

Reply:
Good catch!

@emroller16 the `HStack` in the Basic story should follow the `Modal.Body` (rather than living inside it) and have `spacing="s" padding="xxs" marginTop="xxs"` (just like the `HStack` footers in the rest of the Modal examples).

Reply:
@mannycarrera4 thanks for this! I'm noticing that the basic example also uses `<Box />` inside, while some of the others don't. Should that stay?

Reply:
Box for text is fine, the footer piece should use the HStack!